loom/script/native/core/system/Platform/lmFile.cpp

```diff
@@ -79,6 +79,19 @@ class File
         return 1;
     }
 
+    static int moveFile(lua_State *L)
+    {
+        if (!lua_isstring(L, 1) || !lua_isstring(L, 2))
+        {
+            lua_pushboolean(L, 0);
+            return 1;
+        }
+
+        !platform_moveFile(lua_tostring(L, 1), lua_tostring(L, 2)) ? lua_pushboolean(L, 1) : lua_pushboolean(L, 0);
```

Comment:
Maybe store result in bool then push it so we don't have a crazy ternary monster?